sections/futures/Trade/DelayedOrderConfirmationModal.tsx

```diff
@@ -113,6 +113,10 @@ const DelayedOrderConfirmationModal: FC = () => {
 				label: t('futures.market.user.position.modal.estimated-fill'),
 				value: formatDollars(potentialTradeDetails?.price ?? zeroBN, { isAssetPrice: true }),
 			},
+			{
+				label: 'liquidation price',
```

Comment:
Please fetch the label from the translations file instead of hardcoding it, see line 113 for reference